components/automatic-grid/demo/demo.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="../../grid/dist/grid.production.min.css">
+<link rel="stylesheet" href="../components-theme.css">
+<link rel="stylesheet" href="../style.css">
+
+<style>
+    body {
+        margin: 0;
+        padding: 0;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        background-color: white;
+    }
+
+    .automatic-grid-component {
+        width: 80vw;
+    }
+
+    .guic-row {
+        margin: 0;
+    }
+
+    .guic-automatic-grid-container {
+        border-right: 3px solid black;
+        border-bottom: 3px solid black;
+    }
+
+    .automatic-grid-cell {
+        border-top: 3px solid black;
+        border-left: 3px solid black;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    .box {
+        width: 50px;
+        height: 50px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: cadetblue;
+        color: white;
+        font-size: 1.5rem;
```

Comment:
We can add `user-select: none;` to avoid weird behavior in the browsers like releasing the button and getting stuch in the dragMove event.